utils/gitlog2changelog.py

```diff
@@ -15,7 +16,20 @@
     rev_range = "%s..HEAD" % base
 
 # Execute git log with the desired command line options.
-fin = os.popen("git log --summary --stat --no-merges --date=short %s" % rev_range, "r")
+# uses subproccess instead, more secure.
```

Comment:
This is text which should go to commit message and/or PR description.

Reply:
Will do, I'll make sure that goes in the PR description

Reply:
that was added to the PR description, thanks for the notice